.github/workflows/container.yaml

```
@@ -41,14 +41,12 @@ jobs:
           set -x
           releaseBranchFormat='release-v'
```

tag name contains "v" (e.g. `v0.33.0`) so I think it should be 🤔 

```suggestion
          releaseBranchFormat='release-'
```